tests/models/fastvit/test_modeling_fastvit.py

```diff
@@ -0,0 +1,351 @@
+
+    def test_model(self):
+        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        self.model_tester.create_and_check_model(*config_and_inputs)
+
+    def test_hidden_states_output(self):
+        def check_hidden_states_output(inputs_dict, config, model_class):
+            model = model_class(config)
+            model.to(torch_device)
+            model.eval()
+
+            with torch.no_grad():
+                outputs = model(**self._prepare_for_class(inputs_dict, model_class))
+
+            hidden_states = outputs.hidden_states
+
+            expected_num_stages = 5
+            self.assertEqual(len(hidden_states), expected_num_stages)
+
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+
+        for model_class in self.all_model_classes:
+            inputs_dict["output_hidden_states"] = True
+            check_hidden_states_output(inputs_dict, config, model_class)
+
+            # check that output_hidden_states also work using config
+            del inputs_dict["output_hidden_states"]
+            config.output_hidden_states = True
+
+            check_hidden_states_output(inputs_dict, config, model_class)
+
+    def test_for_image_classification(self):
+        config_and_inputs = self.model_tester.prepare_config_and_inputs()
+        self.model_tester.create_and_check_for_image_classification(*config_and_inputs)
+
+    @slow
+    def test_model_from_pretrained(self):
+        for model_name in FASTVIT_PRETRAINED_MODEL_ARCHIVE_LIST[:1]:
+            model = FastViTModel.from_pretrained(model_name)
+            self.assertIsNotNone(model)
+
+
+# We will verify our results on an image of cute cats
+def prepare_img():
+    image = Image.open("./tests/fixtures/tests_samples/COCO/000000039769.png")
+    return image
+
+
+@require_torch
+@require_vision
+class FastViTModelIntegrationTest(unittest.TestCase):
+    @cached_property
+    def default_feature_extractor(self):
+        return (
+            # TODO update org
+            ViTFeatureExtractor.from_pretrained("jorgeav/fastvit_t8")
+            if is_vision_available()
+            else None
+        )
+
+    @slow
+    def test_inference_image_classification_head(self):
+        # TODO update org
+        model = FastViTForImageClassification.from_pretrained("jorgeav/fastvit_t8").to(torch_device)
+
+        feature_extractor = self.default_feature_extractor
+        image = prepare_img()
+        inputs = feature_extractor(images=image, return_tensors="pt").to(torch_device)
+
+        # forward pass
+        with torch.no_grad():
+            outputs = model(**inputs)
+
+        # verify the logits
+        expected_shape = torch.Size((1, 1000))
+        self.assertEqual(outputs.logits.shape, expected_shape)
+        expected_slice = torch.tensor([-5.6952, -4.7814, -5.7661]).to(torch_device)
+
+        self.assertTrue(torch.allclose(outputs.logits[0, :3], expected_slice, atol=1e-2))
```

Comment:
For other models ([cvt](https://github.com/huggingface/transformers/blob/fc63914399b6f60512c720959f9182b02ae4a45c/tests/models/cvt/test_modeling_cvt), [vit_msn](https://github.com/huggingface/transformers/blob/fc63914399b6f60512c720959f9182b02ae4a45c/tests/models/vit_msn/test_modeling_vit_msn.py), [efficientformer](https://github.com/huggingface/transformers/blob/fc63914399b6f60512c720959f9182b02ae4a45c/tests/models/efficientformer/test_modeling_efficientformer.py), etc), atol is more strict `1e-4`.

Is it possible to make it `1e-4` here as well?

Reply:
We compare some logits output by the original model with the output given by the model we are porting.
So, the `expected_slice` should be the output of the original model.